app/controller/user.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const Controller = require('egg').Controller;
+const moment = require('moment');
```

Comment:
"moment" is not in package.json, seems it's a indirect dependency. BTW, this package is not recommended anymore, the moment official suggests to use "luxon" or "day.js" instead: https://momentjs.com/docs/#/-project-status/